webbpsf/opds.py

```diff
@@ -271,67 +267,78 @@ def tvcircle(radius=1, xcen=0, ycen=0, center=None, **kwargs):
+        tvcircle(radius=5, color="k", linewidth=1)  # , ls='--')
+        tvcircle(radius=30, color="k", linewidth=1)  # 2, ls='--')
         plt.gca().set_xbound(-max_cycles, max_cycles)
         plt.gca().set_ybound(-max_cycles, max_cycles)
 
         y, x = np.indices(abstrans.shape)
-        y -= abstrans.shape[0] / 2.
-        x -= abstrans.shape[1] / 2.
-        r = np.sqrt(x ** 2 + y ** 2) * ps_pixel_size
+        y -= abstrans.shape[0] / 2.0
+        x -= abstrans.shape[1] / 2.0
+        r = np.sqrt(x**2 + y**2) * ps_pixel_size
 
         mask = np.ones_like(self.data)
         mask[np.where(self.amplitude == 0)] = np.nan
         wgood = np.where(self.amplitude != 0)
 
         components = []
-        for i, label in enumerate(['low', 'mid', 'high']):
+        for i, label in enumerate(["low", "mid", "high"]):
             plt.subplot(2, 3, i + 4)
-            if label == 'low':
+            if label == "low":
                 condition = r <= 5
-            elif label == 'mid':
+            elif label == "mid":
                 condition = (r > 5) & (r <= 30)
             else:
-                condition = (r > 30)
+                condition = r > 30
             filtered = trans * condition
 
             inverse = SFT.SFT3(filtered, max_cycles * 2, self.opd.shape[0], inverse=True)
             inverse = inverse[::-1, ::-1]  # I thought SFT did this but apparently this is necessary to get the high freqs right...
 
-            plt.imshow(inverse.real * mask, vmin=(-vmax) / 1000., vmax=vmax / 1000,
-                       cmap=cmap, origin='lower')  # vmax is in nm, but WFE is in microns, so convert
+            plt.imshow(inverse.real * mask, vmin=(-vmax) / 1000.0, vmax=vmax / 1000, cmap=cmap, origin="lower")  # vmax is in nm, but WFE is in microns, so convert
             plt.title(label + " spatial frequencies")
-            rms = (np.sqrt((inverse.real[wgood] ** 2).mean()) * 1000)
+            rms = np.sqrt((inverse.real[wgood] ** 2).mean()) * 1000
 
             components.append(rms)
             plt.xlabel("%.3f nm RMS WFE" % rms)
 
         return np.asarray(components)
 
-    def display_opd(self, ax=None, labelsegs=True, vmax=150., colorbar=True, clear=False, title=None, unit='nm',
-                    pupil_orientation='entrance_pupil',
-                    cbpad=None, colorbar_orientation='vertical',
-                    show_axes=False, show_rms=True, show_v2v3=False,
-                    cmap=None):
-        """ Draw on screen the perturbed OPD
+    def display_opd(
+        self,
+        ax=None,
+        labelsegs=True,
+        vmax=150.0,
+        colorbar=True,
+        clear=False,
+        title=None,
+        unit="nm",
+        pupil_orientation="entrance_pupil",
+        cbpad=None,
+        colorbar_orientation="vertical",
+        show_axes=False,
+        show_rms=True,
+        show_v2v3=False,
+        cmap=None,
+    ):
+        """Draw on screen the perturbed OPD
```

Comment:
...here, where it has taken a def statement and expanded it out so that each parameter is on its own line. 

There must be some kind of logic for this, but it's not apparent to me?!